package.json

```diff
@@ -74,6 +73,7 @@
     "**/d3-color": "^3.1.0",
     "**/@miksu/prettier/minimatch": "^3.0.8",
     "**/request/qs": "^6.11.0",
+    "**/make-fetch-happen": "^11.0.3",
```

Comment:
why did we need to add this resolution?

Reply:
because bumping `make-fetch-happen` can also bump http-cache-semantics
```
=> Found "http-cache-semantics@4.1.0"
info Reasons this module exists
   - "node-gyp#make-fetch-happen" depends on it
   - Hoisted from "node-gyp#make-fetch-happen#http-cache-semantics"
```